modules/ossm-deploying-jaeger.adoc

```diff
@@ -48,10 +50,10 @@ spec:
           type: Memory
 ----
 
-[id="ossm-deploying-jaeger-production_{context}"]
-== Production Jaeger deployment
+[id="ossm-deploying-jaeger-production-min_{context}"]
+== Production Jaeger deployment (minimal)
 
-To use the `production` deployment strategy, set  `spec.addons.jaeger.install.storage.type` to 'Elasticsearch' and specify additional configuration options under `install`.   Or you can create and configure your Jaeger instance and set  `spec.addons.jaeger.name` to the name of the Jaeger instance, for example  `jaeger-production`.
+To use the default settings for the `production` deployment strategy, set  `spec.addons.jaeger.install.storage.type` to 'Elasticsearch' and specify additional configuration options under `install`.  Note that the SMCP only supports configuring Elasticsearch resources and image name.
```

Comment:
Elasticsearch should either be in double quotes or backticks but not single quotes here. Also, do you want the note to be in an admonition block? Or is that too much emphasis?

Reply:
Since there's a comment in the YAML example, I think an actual admonition block might be overkill here.